src/helper-functions/search-members.js

```diff
@@ -0,0 +1,11 @@
+const searchMembers = (membersArr, searchTerm) => {
+  return membersArr.filter(
```

Comment:
Should we be putting a check on searchTerm before proceeding with filtering?
Does it equally work the same with empty string too?

Reply:
Yes, we are putting a check on searchTerm before proceeding with filtering. When it is empty string members array is at its original position. When we type something then it finds the string in the array and we get a filtered array.